cabal.project

```diff
@@ -7,6 +7,7 @@ packages:
   hydra-prelude
   hydra-test-utils
   hydra-tui
+  hydra-cardano-api
```

Comment:
This my other choice of name actually :sweat_smile: ... but we discussed that with @ch1bo this morning, and thought that since this module is really about extra functions which are useful _in the context_ of the Hydra project, it made sense to scope it within that namespace.